### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by 1nsider21**
- [DEMO LINK](https://1nsider21.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://1nsider21.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="moyo-header">
+      <a href="#" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="logo"
```
Comment (irenhh) [inline src/index.html:19]: moyo logo

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
+        >
+      </a>
+      <nav class="moyo-nav">
+        <a href="#" class="nav-links is-active">apple</a>
+        <a href="#" class="nav-links">samsung</a>
+        <a href="#" class="nav-links">смартфоны и телефоны</a>
+        <a
+          href="#"
+          class="nav-links"
+          data-qa="hover">ноутбуки и компьютеры
```
Comment (irenhh) [inline src/index.html:30]:
```suggestion
          data-qa="hover"
        >
          ноутбуки и компьютеры
```

File: src/index.html (hunk 1 above)
Comment (1nsider21) [reply src/index.html]: done

File: src/index.html (hunk 2 above)
Comment (1nsider21) [reply src/index.html]: already reworked =)

**Pull request closed without merging**